identity/common/src/main/resources/schema.sql

```diff
@@ -8,7 +8,8 @@
 
 -- Ref: https://docs.spring.io/spring-security/reference/servlet/appendix/database-schema.html#_user_schema
 CREATE TABLE IF NOT EXISTS users (
-    username varchar not null primary key,
+    id bigint generated by default as identity(start with 0) primary key,
+    username varchar_ignorecase(50) not null unique,
```

Comment:
❌ this will fail for Postgres because `varchar_ignorecase` is not a valid thing there, I'd opt for just `varchar` instead